src/recipe/Baked.Recipe.Service.Application/Runtime/RuntimeLayer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Baked.Architecture;
+using Baked.CodeGeneration;
+using Baked.Domain.Model;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+using static Baked.HttpServer.HttpServerLayer;
+using static Baked.Runtime.RuntimeLayer;
+
+namespace Baked.Runtime;
+
+public class RuntimeLayer : LayerBase<CreateBuilder, BuildConfiguration, AddServices, PostBuild>
+{
+    readonly IServiceCollection _services = new ServiceCollection();
+
+    protected override PhaseContext GetContext(CreateBuilder phase)
+    {
+        var builder = Context.GetWebApplicationBuilder();
+
+        builder.Logging.ClearProviders();
+
+        return phase.CreateContext(builder.Logging);
```

Comment:
get the logging builder from context instead of web application builder.

this means that you need http server layer to add logging builder to the context. i think this is ok, because http server layer already adds service provider in post build anyway

Reply:
`ILoggerBuilder` seems to be used only to provide helper extensions for configuring logger services. Default implementation is only a class with service collection. In order to remove dependency to `HttpServer` layer and introduce additional phases, adding `ILoggerBuilder` in `Runtime` during `AddServices` phase seemed a better solution

Reply:
nice, this is way better